core/mondoo-okta-security.mql.yaml

```diff
@@ -1,379 +1,1985 @@
+    title: Limit the number of super admins
     impact: 100
+    filters: asset.platform == "okta-org"
+    mql: okta.users.where( roles.map(type).contains("SUPER_ADMIN") ).length < props.maxOktaSuperAdmins
+    docs:
+      remediation:
+        - id: console
+          desc: |
+            ### Change admin privileges to a user or an Okta group
+
+            1. In the Admin Console, go to **Security > Administrators**.
+            2. Under **Admin Roles**, select the Super filter to display only super administrators.
+            3. Next to each user entry, click **Edit**. The Edit Administrator window is displayed.
+            4. From the list of administrator roles, assign a role other than super admin to the user.
+            5. Click **Update Administrator**.
+
+  - uid: mondoo-okta-security-threatinsight-block-suspicous-ip-addresses
+    title: Enable Okta ThreatInsight to block suspicious IP addresses
+    impact: 100
+    docs:
+      desc: |
+        Configure network blocklisting to deny access from known malicious IP addresses or locations from your Okta org.
+
+        Admins can block access to their Okta org to IP addresses coming from network zones, IP zones, and dynamic zones. Network zones contain a list of IP addresses, and dynamic zones contain a list of locations, ASNs, or IP types. Okta doesn't allow blocklisted IP addresses to access any of your org's URLs. Okta blocks these requests before any type of policy evaluation occurs.
+
+        ### Okta recommends
+
+        Block any known untrusted IP addresses, locations, or proxy servers to limit access to your org. If your org uses IP Trust for network zones, Okta also recommends blocking any IP addresses that are identified as a Tor anonymizer proxy. Only add IP addresses or locations that aren't associated with legitimate user activity.
+
+        ### Security impact
+
+        Moderate
+
+        ## User impact
+
+        Low
+
+        Legitimate users within your org see no change in behavior. Clients connecting from blocked network zones see a 403 (access denied) error.
+    variants:
+      - uid: mondoo-okta-security-threatinsight-block-suspicous-ip-addresses-terraform-hcl
+      - uid: mondoo-okta-security-threatinsight-block-suspicous-ip-addresses-terraform-plan
+      - uid: mondoo-okta-security-threatinsight-block-suspicous-ip-addresses-terraform-state
+      - uid: mondoo-okta-security-threatinsight-block-suspicous-ip-addresses-runtime
+  - uid: mondoo-okta-security-threatinsight-block-suspicous-ip-addresses-runtime
+    title: Enable Okta ThreatInsight to block suspicious IP addresses
+    impact: 100
+    filters: asset.platform == "okta-org"
+    mql: |
+      okta.networks.any( usage == "BLOCKLIST" && gateways.length > 0 )
+    docs:
+      remediation: 
+        - id: console 
+          desc: |
+            ### Block specific IP addresses
+            Block specific IP addresses to deny access to your Okta org.
```

Comment:
```suggestion
            Block specific IP addresses to deny access to your Okta organization.
```

Reply:
@tas50 these are not our steps. they are from Okta's docs. They are copy/pasted.